tests/integration/test_auth_proxy.py

```diff
@@ -83,7 +83,8 @@ async def is_redirected_to_dex() -> Match[str] | None:
 
     # Choose provider
     async with page.expect_navigation():
-        await page.get_by_role("button", name="Dex").click()
+        # Increase timeout to wait for dex to be ready
+        await page.get_by_role("button", name="Dex").click(timeout=60000 * 5)
```

Comment:
Would timeout be better formatted in minutes i.e. 60 * number_of_seconds?

Reply:
I removed the timeout as it's no longer needed.